pygerm/caproto.py

```diff
@@ -198,6 +198,8 @@ def __init__(self, *, zclient,
                     fname = path / '{}.h5'.format(str(uuid.uuid4()))
                     with h5py.File(str(fname), 'w-') as fout:
                         g = fout.create_group('GeRM')
+                        # NOTE : this usage is now deprecated
+                        # Need to update if this class is used again
```

Comment:
Can you make this raise instead?

Reply:
this should not have been necessary. removed